numpy/core/setup.py

```diff
@@ -1005,6 +1005,7 @@ def generate_umath_doc_header(ext, build_dir):
             join('src', 'umath', 'loops.h.src'),
             join('src', 'umath', 'loops_utils.h.src'),
             join('src', 'umath', 'loops.c.src'),
+            join('src', 'umath', 'loops_unary.dispatch.c.src'),
```

Comment:
A similar addition is needed for the meson.build just above this line https://github.com/numpy/numpy/blob/7f0f045625022c3f816911cd80f8635ac2a36f21/numpy/core/meson.build#L750